### PR TITLE
VC-4248 system transfer

### PR DESCRIPTION
**Pull request by alex-bodnar**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: database/schema/14-banking.sql [hunk 1]
```diff
@@ -68,12 +40,70 @@ CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_issue (
         REFERENCES vipcoin_chain_banking_base_transfers(id)
 );
 
+CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_msg_system_transfer (
+    creator             TEXT       NOT NULL,                    -- banking creator
+    wallet_from         TEXT       NOT NULL,                    -- banking wallet from
+    wallet_to           TEXT       NOT NULL,                    -- banking wallet to
+    asset               TEXT       NOT NULL,                    -- banking asset
+    amount              BIGSERIAL,                              -- banking amount
```
Comment (DirusK) [inline database/schema/14-banking.sql]: Нам не не нужно это автоинкрементировать
Bigint может

Comment (DirusK) [reply database/schema/14-banking.sql]: Так же с остальными amount

File: database/types/vipcoin_banking.go [hunk 2]
```diff
@@ -42,4 +42,65 @@ type (
...
 		Wallet string `db:"wallet"`
 	}
+
+	// DBSystemTransfer represents a single row inside the "vipcoin_chain_banking_msg_system_transfer" table
+	DBMsgSystemTransfer struct {
+		Creator    string  `db:"creator"`
+		WalletFrom string  `db:"wallet_from"`
+		WalletTo   string  `db:"wallet_to"`
+		Asset      string  `db:"asset"`
+		Amount     uint64  `db:"amount"`
```
Comment (DirusK) [inline database/types/vipcoin_banking.go:52]: в теории могут быть проблемы когда amount выйдет за диапазон
у нас и так 8 чисел добавляется к коину, а uint всего 19 хранит
Может decimal использовать или sdk коин если он может, хз

Comment (alex-bodnar) [reply database/types/vipcoin_banking.go:52]: В chain так заложено, я оттуда брал типы

Comment (DirusK) [reply database/types/vipcoin_banking.go:52]: в чейне же Coins вроде юзается или про что ты?

Comment (alex-bodnar) [reply database/types/vipcoin_banking.go:52]: `
type BaseTransfer struct {
	Id        uint64         `protobuf:"varint,1,opt,name=id,proto3" json:"id,omitempty"`
	Asset     string         `protobuf:"bytes,2,opt,name=asset,proto3" json:"asset,omitempty"`
	Amount    uint64         `protobuf:"varint,3,opt,name=amount,proto3" json:"amount,omitempty"`
	Kind      TransferKind   `protobuf:"varint,4,opt,name=kind,proto3,enum=vipcoin.chain.banking.TransferKind" json:"kind,omitempty"`
	Extras    []*types.Extra `protobuf:"bytes,5,rep,name=extras,proto3" json:"extras,omitempty"`
	Timestamp int64          `protobuf:"varint,6,opt,name=timestamp,proto3" json:"timestamp,omitempty"`
	TxHash    string         `protobuf:"bytes,7,opt,name=tx_hash,json=txHash,proto3" json:"tx_hash,omitempty"`
}`

**Comment by DirusK** [comment]
LGTM

**Review by IamStubborN — APPROVED** [review]
LGTM

**Pull request merged**